### DISCUSSION TITLE
Label - Figure's name

### DISCUSSION THREAD

**Discussion by RBstudentPhD** [mtex-toolbox/mtex#1386] [Ask Anything]
Hi,

I have a non-technical question, but more a question concerning the crystallography vocabulary. 
I'm trying to find the appropriate name for the following figure (ODF along the X-axis of an EBSD in the hexagonal IPF triangle) :
```
ebsd = loadEBSD_ctf(ctf_file_name,'convertEuler2SpatialReferenceFrame');
    % The crystal symmetry is Titanium
    ODF = unimodalODF(ebsd.orientations);
    figure; plotIPDF(ODF,xvector,'antipodal') 
    mc=mtexColorbar('location','eastoutside','title','ODF','FontSize',FS); 
    saveFigure('ODF_example')
```
<img width="684" alt="ODF_example" src="https://user-images.githubusercontent.com/100238542/159071159-090c37fc-e9be-4afa-88ae-907fb58222c2.png">

I named the colorbar 'ODF' but my PhD director told me it was wrong, but I didn't find it in the literature.
Also, what title would you give to that figure in an article?
Anyone could help me with that, please?

Thank you in advance,
Cordially,
RB

**Comment by lfgmorales** [comment]
Hi,



If you are only talking about the scale of your inverse pole figure, it is given in multiples of uniform distribution (MUD), where MUD with a value of 1 is exactly UNIFORM



cheers



Luiz




________________________________
From: RBstudentPhD ***@***.***>
Sent: 18 March 2022 20:29
To: mtex-toolbox/mtex
Cc: Subscribed
Subject: [mtex-toolbox/mtex] Label - Figure's name (Discussion #1386)


Hi,

I have a non-technical question, but more a question concerning the crystallography vocabulary.
I'm trying to find the appropriate name for the following figure (ODF along the X-axis of an EBSD in the hexagonal IPF triangle) :

ebsd = loadEBSD_ctf(ctf_file_name,'convertEuler2SpatialReferenceFrame');
    % The crystal symmetry is Titanium
    ODF = unimodalODF(ebsd.orientations);
    figure; plotIPDF(ODF,xvector,'antipodal')
    mc=mtexColorbar('location','eastoutside','title','ODF','FontSize',FS);
    saveFigure('ODF_example')


[ODF_example]<https://user-images.githubusercontent.com/100238542/159071159-090c37fc-e9be-4afa-88ae-907fb58222c2.png>

[https://user-images.githubusercontent.com/100238542/159071159-090c37fc-e9be-4afa-88ae-907fb58222c2.png]

I named the colorbar 'ODF' but my PhD director told me it was wrong, but I didn't find it in the literature.
Also, what title would you give to that figure in an article?
Anyone could help me with that, please?

Thank you in advance,
Cordially,
RB

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1386>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5INZC5AOVK5SWMBTAESTVATKQNANCNFSM5RCW6ZDQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1386", "url": "https://github.com/mtex-toolbox/mtex/discussions/1386", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]
